app/enterprise/2.1.x/kong-for-kubernetes/install.md

```diff
@@ -11,7 +11,7 @@ Kong for Kubernetes Enterprise provides most Kong Enterprise plugins and runs wi
 
 >Note: See [Kong for Kubernetes deployment options](/enterprise/{{page.kong_version}}/kong-for-kubernetes/deployment-options) for a feature breakdown and image comparison.
 
-You can install Kong for Kubernetes Enterprise using YAML with kubectl, or with OpenShift oc. Other deployment options, such as using Helm Chart and Kustomize, will be available at a later time.
+You can install Kong for Kubernetes Enterprise using YAML with kubectl, with OpenShift oc, or [with Helm](https://github.com/Kong/charts/tree/master/charts/kong).
 
 <img src="https://doc-assets.konghq.com/kubernetes/K4K8S-Enterprise-Diagram.png" alt="Kong for Kubernetes Enterprise control diagram">
```

Comment:
Is this diagram still accurate? 
(Was this diagram accurate before...)

Reply:
Yeah, it's fine--this diagram is very simple and describes functionality, which isn't changing. Implementation details are different, but the functionality is the same at a high level.